app/assets/javascripts/covid19/src/webapp/components/app/Root.tsx

```diff
@@ -16,10 +16,12 @@ export const Root: React.FC = React.memo(() => {
     const [title, setTitle] = React.useState<React.ReactNode>(<></>);
     const [isAccordionExpanded, { toggle: toggleAccordion }] = useBooleanState(false);
     const [visible, setVisible] = React.useState<VisibleGen>({});
+    const [detailsVisible, setDetailsVisible] = React.useState(false);
```

Comment:
Not sure the parent needs to have control over this. In that case, the state can be within SVGProteome.

Reply:
I did cause when the user clicks the hide button, the title was expected to change to the default one. So if the details section is not hidden, having the default title with 'X' details could be misleading.